src/vm/exceptionhandling.cpp

```diff
@@ -5168,6 +5171,28 @@ void CleanUpForSecondPass(Thread* pThread, bool fIsSO, LPVOID MemoryStackFpForFr
     }
 }
 
+#ifdef FEATURE_PAL
+
+EXTERN_C _Unwind_Reason_Code
+UnhandledExceptionHandlerUnix(
```

Comment:
Could you please add a header for this function with a description where it is used and why we need it since it is not clear from the name of the function that it is not used for managed exceptions?